README.md

```diff
@@ -24,6 +24,12 @@ Please note: We take Terraform's security and our users' trust very seriously. I
     - Note that version 0.11.x currently works, but is [deprecated](https://www.hashicorp.com/blog/deprecating-terraform-0-11-support-in-terraform-providers/)
 -	[Go](https://golang.org/doc/install) 1.14.x (to build the provider plugin)
 
+
+## Kubernetes Alpha Provider
+
+An new [experimental provider](https://github.com/hashicorp/terraform-provider-kubernetes-alpha) is now available that enables management of all Kubernetes resources, including CustomResourceDefinitions (CRDs). Our intent is to eventually merge these two providers.
```

Comment:
LGTM, just a type "An new" -> "A new". Thanks Tony

Reply:
Should we put in a link to the registry instead of github?

Reply:
@aareet I thought about that, but since this is on the GH readme, felt more natural to point to the other repo. I'm all for driving traffic to the registry though, if you're good w/ it I am :D